en/docs/reference/connectors/pulsar-connector/pulsar-connector-example.md

```diff
@@ -0,0 +1,260 @@
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Before you begin
+
+To connect with Apache Pulsar using the WSO2 Micro Integrator Apache Pulsar Connector, you need to first set up a running Pulsar instance locally or on a server. In this example, we will use an Apache Pulsar standalone server. Follow the steps below to prepare the Pulsar environment:
+
+### 1. Download and Extract Pulsar
+
+- Download the latest Apache Pulsar release from the [official website](https://pulsar.apache.org/download/).
+- Extract the archive to a preferred location on your machine.
+
+---
+
+### 2. Configure TLS Encryption
+
+- For this example, **TLS encryption using PEM certificates** is used to ensure secure communication with the Pulsar broker. To configure TLS with PEM, refer to the official [Apache Pulsar documentation](https://pulsar.apache.org/docs/next/security-tls-transport/#configure-mtls-encryption-with-pem) for a step-by-step guide and ensure that you have the following components prepared:
+
+    - A **Certificate Authority (CA)** certificate - `tlsTrustCertsFilePath`
+    - A **server certificate** - `tlsCertificateFilePath`
+    - The **server's private key** - `tlsKeyFilePath`
+
+---
+
+### 3. Configure Authentication using tokens based on JSON Web Tokens (JWT)
+
+- For this example, **JWT-based authentication** is used to restrict access to authorized clients only. To configure JWT authentication, refer to the official [Apache Pulsar documentation](https://pulsar.apache.org/docs/next/security-jwt/) for a step-by-step guide. Ensure that you have the following components prepared:
+
+    - A **JWT token** that includes the necessary claims for authentication. The compact representation of a signed JWT is a string that looks like:
+        ```bash
+        eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
+        ```
+  
+    Once the JWT token is generated, configure the Pulsar broker to enable JWT-based authentication.
+
+### 4. Start the Pulsar Standalone Server
+
+- Navigate to the extracted Pulsar directory.
+- Start the standalone server using the following command:
+
+      ```bash
+      bin/pulsar standalone
+      ```
+
+!!!Note
+    - For setting up Pulsar in other environments such as Docker or Kubernetes, please refer to the [official Apache Pulsar documentation](https://pulsar.apache.org/docs/next/getting-started-home/).
+    - The recommended version of Java for Apache Pulsar is 21 or above.
+
+### Setup Environment for Apache Pulsar Connector
+
+To configure the Apache Pulsar connector with Apache Pulsar version 4.0.4, copy the following client libraries to the `<MI_HOME>/lib` directory.
+
+* [pulsar-client-4.0.4.wso2v1.jar](https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.12/2.8.2)
```

Comment:
Won't these dependencies get packed by the connector itself when creating the CAR?